pkg/session_recording/recordings.jsx

```diff
@@ -249,7 +254,7 @@
                     <div className="row">
                         <div className="col-md-12">
                             <ol className="breadcrumb">
-                                <li><a>Session Recording</a></li>
+                                <li><a onClick={this.goBackToList}>Session Recording</a></li>
```

Comment:
No, it was not working. I took jquery as an example from other place, but actually the way I did it is more reliable.